### DISCUSSION TITLE
How did you create the webp in the README?

### DISCUSSION THREAD

**Discussion by alkasm** [ajeetdsouza/zoxide#420] [Q&A] [answered]
Sorry for asking questions outside the scope of the project, but I've been trying to similarly record useful demos from my terminal for projects. My best attempt was hacking the heck out of the timestamps for a recorded session from `asciinema`, which worked but was a huge amount of work. Would like to know what your method was!

**Comment by ajeetdsouza** [comment, marked as answer]
Hi @alkasm, glad you liked the demo! I too did it using asciinema. I just typed it out really slowly so that I didn't make a mistake, then played it back at a higher speed while recording it with a screen capture software.
